src/AddCard/components/AddCardForm/components/CardNumber/hooks/useControlledCardNumber.ts

```
@@ -0,0 +1,45 @@
+import { useCallback, useState } from "react";
+import { isNaN } from "../../../../../../utils/isNaN";
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)